hpcgap/lib/vecmat.gd

```diff
@@ -431,6 +431,34 @@ DeclareOperation( "ImmutableMatrix",[IsObject,IsMatrix]);
 
 #############################################################################
 ##
+#F  ImmutableVector( <field>, <vector>[, <change>] ) . convert into "best" representation
+##
+##  <#GAPDoc Label="ImmutableVector">
+##  <ManSection>
+##  <Oper Name="ImmutableVector" Arg='field, vector[, change]'/>
+##
+##  <Description>
+##  returns an immutable vector equal to <A>vector</A> which is in the optimal
+##  (concerning space and runtime) representation for vectors defined over
+##  <A>field</A>. This means that vectors obtained by several calls of
+##  <Ref Oper="ImmutableVector"/> for the same <A>field</A> are compatible
+##  for fast arithmetic without need for field conversion.
+##  <P/>
+##  The input vector <A>vector</A> might change the representation,
```

Comment:
Does it become clearer if I change it to "might change its representation"?

(Not an excuse, but: this text is also copied from that for `ImmutableMatrix` ;-). So if we tweak it here, we should also tweak it there).